app/models/manageiq/providers/oracle_cloud/regions.rb

```diff
@@ -2,7 +2,8 @@ module ManageIQ
   module Providers::OracleCloud
     class Regions < ManageIQ::Providers::Regions
       private_class_method def self.from_source
-        require "oci"
+        require "oci/common"
```

Comment:
`"oci/common"` only uses 7.2MiB and is enough to access the `OCI::Regions::REGION_ENUM` (NOTE `require "oci/regions"` does not work there is more setup needed before that can be accessed)